### ISSUE TITLE
Errors installing tekton resources defined in default collection

### ISSUE THREAD

**Issue by kaczyns** [kabanero-io/kabanero-operator#79]
There appears to be an ordering and/or timing problem when creating the default collection. Sometimes, `Pipeline` and `Task` resources cannot be created by the collection controller, presumably because the Tekton operator has not finished processing the `Install` resource yet (this causes Tekton to register the `Pipeline` and `Task` CRDs).  The error can happen in several ways, some of which include (from the Kabanero operator log):
```
"error":"no matches for kind \"Task\" in version \"tekton.dev/v1alpha1\""
```
or
```
"error":"the server could not find the requested resource"
```
Eventually, the CRDs are registered, but the operator does not seem to always detect this, and so future loops of the collection controller continue to receive the error.  We should investigate whether we can fix the timing here... perhaps if the Kabanero controller is going to create a `Install`, it could wait until the `Pipeline` and `Task` CRDs are available before trying to create the `Collection` instance.  It may be that the particular collection does not define a `Pipeline` or `Task`, but that's a harder problem to solve.

**Comment by kaczyns** [comment]
This may all be a moot point after #71 is resolved, if we move to pipelines 0.5.  The install process appears to be different and is done at the cluster level.

**Comment by kaczyns** [comment]
With issue #71 now resolved, the tekton.dev CRDs should be available well before a `Kabanero` instance is created, and the problem reported here should not occur.
